.eslintrc.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
     'no-new': 'off',
     'jest/no-standalone-expect': 'off',
     'node/no-callback-literal': 'off',
-    'unused-imports/no-unused-imports': 'error'
+    'unused-imports/no-unused-imports': 'error',
+    'no-useless-constructor': 0
```

Comment:
still needed? I see no empty args constructors in the diff

Reply:
In general, please don't add to the eslint config, but instead disable checks via comment for the one line where it's needed.

Reply:
oh i think this was by mistake